Tests/NautilusTelemetryTests/Utilities/FlushTimerTests.swift

```diff
@@ -44,32 +42,24 @@ final class FlushTimerTests: XCTestCase {
 			}
 		}
 
-		// Wait for first firing
 		wait(for: [expectation1], timeout: 1.0)
 		XCTAssertEqual(handlerCallCount, 1)
 
-		// Change interval and verify it takes effect
+		// Check minimum enforced
 		timer.flushInterval = 0.05
-		XCTAssertEqual(timer.flushInterval, 0.05)
+		XCTAssertEqual(timer.flushInterval, timer.minimumFlushInterval)
```

Comment:
```suggestion
        let tooSmallFlushInterval = 0.05
        XCTAssertNotEqual(tooSmallFlushInterval, timer.minimumFlushInterval)
        timer.flushInterval = tooSmallFlushInterval
		XCTAssertEqual(timer.flushInterval, timer.minimumFlushInterval)
```

This would be a bit more powerful and robust to changes.